nix/rocksdb.nix

```diff
@@ -21,13 +21,13 @@
 
 stdenv.mkDerivation rec {
   pname = "rocksdb";
-  version = "8.6.7";
+  version = "8.1.1";
```

Comment:
do we need replace grocksdb to v1.8.0 as well

Reply:
yeah, we need